Topics/jupyter-notebook.md

```diff
@@ -0,0 +1,154 @@
+# Jupyter Notebook
+
+# What you will learn 
+In this guide you'll learn how to use the Jupyter Notebook and the integrated JupyterLab IDE. Jupyter is a free and open-source web application (in the form of a Python module) that allows you to create and share documents that contain code, equations, text and images incorporated.
+
+It's widely used in the Data Science process, in order to explore data, clean them, and experiment problem solutions.
+
+## Prerequisites
+- [Python Fundamentals](python-fundamentals.md)
+
+## Time to complete
+5 minutes.
+
+
+# Index
+ - [Why Jupyter Notebooks](#Why-Jupyter-Notebooks)
+ - [Learn to use it](#Learn-to-use-it)
+ - [Considerations](#Considerations)
+ - [Tips and Tricks](#Tips-and-Tricks)
+ - [Notebooks Examples](#Notebooks-Examples)
+ - [Conclusions](#Conclusions)
+
+
+Let's dive right in!
+ 
+### Why Jupyter Notebooks
+
+[Jupyter notebooks](https://jupyter.org/) are a way of embedding ready-to-execute python code with markdown documentation, images, and other such things, all in one file, as well as a program for presenting and running them. 
+
+They're nice if you want to demonstrate a short script to another person, particularly in a data science context, because of some built-in plotting niceties and the fact that you can run cells of code individually. 
```

Comment:
And they save the results into the document, even after you have stopped running the Jupyter server that hosts all the notebooks -- must say this. 

Most services ie. GitHub recognise the .ipynb  file extension and display notebook contents in the GUI - so such save results can be seen, even though static can be useful for many purposes like presentation or sharing online.

Reply:
done